content/applications/websites/ecommerce/managing_products/products.rst

```diff
@@ -0,0 +1,197 @@
+========
+Products
+========
+
+Odoo allows you to create, import, and manage your products' pages all within the **Website** app.
+
+Add products to the catalog
+===========================
+
+To add a product to your catalog, you can either do it in:
+
+- The front end: from the :guilabel:`Homepage`, click :guilabel:`+ New` in the top-right corner, and
+  select :guilabel:`Product`. Enter the name of your product, and :guilabel:`Save`;
+- The back end: by going to :menuselection:`eCommerce --> Products --> Create`. You can find more
+  information on product creation :doc:`here <../../../sales/sales>`;
+- By importing: you can :ref:`import data <import-data>` using XLSX or CSV files.
+
+.. seealso::
+   - :doc:`../../../sales/sales/products_prices/products/import`
+
+Publish
+-------
+
+Upon creation, products are defaulted as :guilabel:`Unpublished` in your eCommerce catalog.
+**Unpublished products** are only visible to users with access rights to your eCommerce, whereas
+:guilabel:`Published` products are visible to all users. When you wish to render a product visible
+to all users, go to :menuselection:`Site --> Homepage --> Shop`, select the product, and enable the
```

Comment:
> by default, we are on the Homepage, isn't it? In this case, I would remove the "Site --> Homepage" or replace "site" by "website app"

You're not on the "Homepage" by default, depending on where you are during the flow. You could be anywhere but the homepage.

Reply:
> by default, we are on the Homepage, isn't it? In this case, I would remove the "Site --> Homepage" or replace "site" by "website app"

By default we are on the homepage, but depending on where in the flow the user is when reading this, I prefer to leave in the full info to avoid ambiguity and misconfusion